### ISSUE TITLE
Questions about running the code.

### ISSUE THREAD

**Issue by suoych** [lukashermann/hulc#6]
Hi, thanks for sharing your work, I met this problem when I followed the instructions run the training code.

![image](https://user-images.githubusercontent.com/29878050/202217223-98a30dcc-14db-40fc-88ee-69998296e3ed.png)
I successfully installed the calvin_env.
![image](https://user-images.githubusercontent.com/29878050/202217493-2fb85a4e-1c9c-4825-9df3-9a95f7b5cf12.png)

I don't know why this happens, there might be something wrong with the calvin_env?
It would be so nice if you could answer my question, thanks again.

**Comment by lukashermann** [comment]
Does the same error occur if you import it directly in a python console?
```
from calvin_env.envs.tasks import Tasks
```

**Comment by suoych** [comment]
Yeah I tried before and it says there is no module named  calvin_env.envs

@lukashermann 
> Does the same error occur if you import it directly in a python console?
> 
> ```
> from calvin_env.envs.tasks import Tasks
> ```

**Comment by lukashermann** [comment]
And if you run `pip list` it shows `calvin_env`?
I don't know what could be the problem.

**Comment by suoych** [comment]
> And if you run `pip list` it shows `calvin_env`? I don't know what could be the problem.

Yeah, it shows calvin_env, pip show calvin_env is also available.
I'll try to figure it out by myself then, thanks for your patience anyway.

**Comment by lukashermann** [comment]
What happens if you only do `import calvin_env`? 
I guess the problem is unrelated to our code since you are the first to spot this error, but keep me updated if you find the solution!

**Comment by suoych** [comment]
@lukashermann  Hi, I found the problem. After I installed the calvin_env, I ran the training code in the root directory of hulc project, which will cause the "import" command in python to import the calvin_env folder first instead of the installed lib. 

The solution is to run the training code in another directory or install the calvin_env lib in another directory.

Sorry for my negligence and thanks again for your reply.

> What happens if you only do `import calvin_env`? I guess the problem is unrelated to our code since you are the first to spot this error, but keep me updated if you find the solution!
